layouts/partials/footer.html

```diff
@@ -43,7 +43,7 @@ <h5>{{ i18n "navigation" }}</h5>
         <h5>{{ i18n "contact_me" }}</h5>
         <ul>
           {{ range $key,$value:= $author.contactInfo }}
-          <li><span>{{ title $key }}: </span> <span>{{ $value }}</span></li>
+          <li><span>{{ i18n $key }}: </span> <span>{{ $value }}</span></li>
```

Comment:
We don't need i18n translation here. The author information will be on their own locale (i.e. `en`/`fr` folder). So, they will be already translated there.